Makefile

```diff
@@ -88,25 +88,31 @@ test: test-production test-presentation
 
 # run synchronous (without ActiveMQ)
 test-production: SCRIPT = process_images.sh --proc-id 1 --lang deu --script Fraktur
+test-production: MANAGER_PS != docker container ls -n1 -qf name=ocrd-manager
```

Comment:
Can we name it consistently? `MANAGER_PS` =>  `MANAGER_CONTAINER` cause `test-kitodo` or integration repo use this name convention and `MANAGER_PS` is misleading IMO. Or we use in both cases a more general designation like `CONTAINER` cause `NETWORK` and `DATA` is also more general.